package.json

```diff
@@ -0,0 +1,82 @@
+{
+  "name": "todo",
+  "version": "1.0.0",
+  "description": "sample todo app for typescript",
+  "main": "index.js",
+  "scripts": {
+    "build": "gulp build",
+    "start": "node ./dist/server/server",
+    "lint": "tslint --fix -c tslint.json src/**/*.ts test/**/*.ts",
+    "heroku-postbuild": "npm run build",
+    "heroku-start": "node ./dist/server/server",
+    "test": "NODE_ENV=testing mocha --exit -r ts-node/register --project tsconfig.json test/**/*.spec.ts",
+    "serve": "gulp serve"
+  },
+  "dependencies": {
+    "@sendgrid/mail": "^7.1.0",
```

Comment:
Remove unwanted packages from `dependencies ` and `devDependencies `

Reply:
Removed unnecessary dependencies like sendgrid and bcrypt